back/engines/commercial/project_folders/app/models/project_folders/folder.rb

```diff
@@ -45,6 +47,10 @@ class Folder < ::ApplicationRecord
 
     after_destroy :remove_moderators
 
+    pg_search_scope :search_by_all,
+      against: %i[title_multiloc description_multiloc description_preview_multiloc slug],
```

Comment:
Searching against jsonb columns means we are going to match queries by the JSON keys too, right? Are we sure we want this?

A column entry such as:

```
# title_multiloc
{
  en: "Fancy title",
  de: "Cooler Titel
}
```

would match against the search query 'en', even if htat's not part of the title.

Reply:
Yeah, I think it's not perfect at the moment, but probably "good enough" for a first revision. You would also get results from other languages and be unable to see the content (if your app is in English, you search "en", and the French translations have "en" in them, the project would come up but you wouldn't necessarily see why).

I think generally from what I know of it, the number of projects on even the biggest platforms is small enough that any reasonably concise search term ("park", "downtown") will only bring up a handful of projects, so I think the current state (while not super exact) is decent. It would be good to keep track of this feature though and see if/how users are actually using it

Reply:
Ok, then let's proceed with the current status quo! :)